api-client/src/main/java/com/xing/api/Resource.java

```diff
@@ -17,11 +17,7 @@
 
 import java.lang.reflect.Type;
 
-/**
- * TODO docs.
- *
- * @author serj.lotutovici
- */
+/** Base class for all resources providing the {@linkplain CallSpec.Builder} to build a network operation. */
```

Comment:
better would be:
`Base class for all resources that provide access to various XING api endpoints.`